dbt_coves/tasks/setup/ssh.py

```diff
@@ -32,25 +33,58 @@ def register_parser(cls, sub_parsers, base_subparser):
     @classmethod
     def run(cls) -> int:
         ssh_status = "[red]MISSING[/red]"
-        key_path = "~/.ssh/id_rsa"
+        key_path = "~/.ssh/id_ecdsa"
```

Comment:
To check if a ssh key exists, we should consider all flavours, not just the ones generated with `ecdsa` alg.